src/main/java/be/ephec/java/exercice/Main.java

```diff
@@ -1,7 +1,64 @@
 package be.ephec.java.exercice;
 
+/*Même si dans le même package, la classe Person n'est pas visible depuis Main,
+j'ai dû l'importer pour faire l'exercice*/
+
+import be.ephec.java.exercice.e01.Person;
+import java.time.LocalDate;
+import java.util.Scanner;
+
 public class Main {
     public static void main(String[] args) {
-        System.out.println("Hello world!");
+        //E01:
+        Scanner keyboard = new Scanner(System.in);
+        Person person = new Person();
+        System.out.println("What is your first name?");
+        person.firstName = keyboard.next();
+        System.out.println("What is your name");
+        person.lastName = keyboard.next();
+        System.out.println("What is your birth year?");
+        person.yearOfBirth = keyboard.nextInt();
+        // J'utilise ici l'année actuelle comme int:
+        person.computeAge(LocalDate.now().getYear());
+        person.display();
+        /////////////////////////////////////////////////
+
+
+        Base.display();
```

Comment:
Comme déjà mentionné plus haut, déplacez ceci dans e02... :)

Reply:
Je pensais que le package était le grand, be.ephec.java.exercice, et que e01, e02, etc étaient des modules. 
Je ne comprends pas très bien la différence, d'ailleurs. J'ai juste vite lu un peu sur internet. Quand on utilise l'un ou l'autre? Est-ce que dans chaque module il faut créer une classe main?
Et est-ce que les "dossiers" e01, e02 sont des packages ou des modules?